src/@next/Banner/FixedBannerStyle.ts

```diff
@@ -76,23 +78,30 @@ export const StyledFixedBannerContentContainer = styled.div`
   padding-right: 16px;
 
   @media (max-width: ${Breakpoints.large}) {
-    position: absolute;
-    top: 16px;
-    left: 52px;
+    padding-left: 36px;
+    padding-right: 0px;
 
-    &[data-noicon='true'] {
-      left: 16px;
+    &[data-dismissable='true'] {
+      padding-right: 36px;
+    }
+
+    &[data-has-action='true'] {
+      padding-bottom: 48px;
+    }
+
+    &:not([data-show-icon='true']) {
+      padding-left: 0px;
     }
   }
 `;
 
 export const StyledFixedBannerButtonContainer = styled.div`
```

Comment:
hi @michac789 I don't think we can set the position of the ButtonContainer to absolute, because we do not know how much content we are going to display in a fixed banner and the content will inevitably break into multiple lines on small screens. On top of that, we would like to see a 16px gap between the text and buttons. Otherwise, we will see an overlap like this in the screenshot.
![image](https://github.com/glints-dev/glints-aries/assets/12844836/94533a98-ab85-4e71-99e1-450345f7bb9f)

Reply:
Let me check... so my idea was that the height of the banner depends on the height of text (add padding bottom for the text container), then put the botton with absolute position from the bottom of the banner (not top)... let me check, i think it worked previously

Reply:
hmm for me it is displayed properly in storybook:
![image](https://github.com/glints-dev/glints-aries/assets/67429929/dfab56e4-a80d-475a-928a-5d2b50f7fdd8)


Haven't changed anything (still using the storybook link that I created last Monday):
![image](https://github.com/glints-dev/glints-aries/assets/67429929/3c31d58b-7a0e-44df-bb7b-08c20404cd23)


Can you check if you're referring to the correct storybook link?
(Storybook Preview: https://615abff875b8f5004aa277ac-fybmupupdy.chromatic.com/) -> based on my latest commit last Monday (05 June 2023)


I think it should work on small screen because:
- Banner has min-height, if the content is large then the height depends on the content (description), I added padding-bottom of 48px if there is an action (instead of 12px because the extra 36px is for the button; note that there is still 16px padding on the banner container itself, so the distance between the bottom border of content div to the bottom border of the banner div is (12 + 36 + 16)px)
- The action is located 16px absolute from the bottom of the page, it should take the extra 36px I have left 

![image](https://github.com/glints-dev/glints-aries/assets/67429929/642cf7f1-b220-4325-b4a8-60659f9a0b73)

Please verify and lmk if somehow it doesn't work on your side!

Reply:
somehow it works well on my end now, but could you help to increase the padding bottom by 4px, as the gap should be 16px instead of 12px @michac789 thanks!

Reply:
@chiahou done